config/100-rolebinding.yaml

```diff
@@ -102,6 +102,8 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: tekton-chains-controller
+  annotations:
+    iam.gke.io/gcp-service-account: chuangw@chuangw-test.iam.gserviceaccount.com
```

Comment:
remove.

Reply:
Sorry. Forgot deleting it. I changed it for testing locally 😂